### DISCUSSION TITLE
School project

### DISCUSSION THREAD

**Discussion by PietervHa** [codinasion-archive/codinasion-monorepo#5738] [General]
Hello guys, for a school project i need to make a contribution on a open source system. Can I help with something? Also what are the programming standards?

**Comment by harshraj8843** [comment]
Hey @PietervHa 

The contribution workflow is as follows

1. Choose any `good first issue` you want to contribute to. [Here is the list of all good first issues not assigned to anyone](https://github.com/codinasion/codinasion/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee+label%3A%22good+first+issue%22+)
2. Comment `!assign` on any `good first issue` to be auto-assigned
3. Create Pull Request (PR) with your solution

> [!NOTE]
> For more info, read our [CONTRIBUTING.md](https://github.com/codinasion/codinasion/blob/master/CONTRIBUTING.md)
